aws-lambda-java-serialization/pom.xml

```diff
@@ -31,47 +32,49 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <relocation.prefix>com.amazonaws.lambda.thirdparty</relocation.prefix>
+        <jackson.version>2.12.6.1</jackson.version>
```

Comment:
Suggestion: Create Parent POM with `dependencyManagement`. To act as BOM

Reply:
Conversation going on here https://github.com/aws/aws-lambda-java-libs/issues/327